membership-attribute-service/app/models/Attributes.scala

```diff
@@ -19,14 +17,36 @@ object ContentAccess {
   implicit val jsWrite = Json.writes[ContentAccess]
 }
 
+case class CardDetails(last4: String, expirationMonth: Int, expirationYear: Int, forProduct: String) {
+  val asLocalDate = new LocalDate(expirationYear, expirationMonth, 1).plusMonths(1).minusDays(1)
```

Comment:
I'd have a slight preference for this to be a `def` rather than a `val` (same on line 31)